libs/angular-components/community/components/overlay/modal/header/modal-header.component.scss

```diff
@@ -0,0 +1,31 @@
+@use "@tehik-ee/tedi-core/mixins";
+
+.tedi-modal-header {
+  --_modal-header-button-color: var(--neutral-900);
+  --_modal-header-button-size: var(--dimensions-14);
+  --_modal-heading-font-size: var(--dimensions-12);
+
+  border-bottom: var(--_modal-border);
+
+  @include mixins.responsive-styles(padding-left, modal-heading-padding-x);
+  @include mixins.responsive-styles(padding-right, modal-heading-padding-x);
+
+  @include mixins.responsive-styles(padding-top, modal-heading-padding-y);
+  @include mixins.responsive-styles(padding-bottom, modal-heading-padding-y);
+
+  &__title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  h1 {
```

Comment:
there's also tedi-text--h3 class added in the template file, doesn't that already add the necessary styles?